src/main/java/org/springframework/samples/petclinic/Pet/PetTypes.java

```diff
@@ -0,0 +1,28 @@
+package org.springframework.samples.petclinic.Pet;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@Getter
+@RequiredArgsConstructor
+public enum PetTypes {
+
+	CAT("cat", "pet.cat"), DOG("dog", "pet.dog"), LIZARD("lizard", "pet.lizard"), SNAKE("snake", "pet.snake"),
+	BIRD("bird", "pet.bird"), HAMSTER("hamster", "hamster.bird");
+
+	private final String value;
+
+	private final String frontendTranslationKey;
+
+	// TODO: Refactor so we will use this instead of taking all Types from db
+	public static List<String> getPetTypeNames() {
```

Comment:
I forgot to use the translation keys in frontend, but can be a TODO for another PR.